### ISSUE TITLE
Recommanded way to import/export encoder profiles ?

### ISSUE THREAD

**Issue by pokemaster974** [staxrip/staxrip#1494]
Hi,
with the last update 2.42.0 (and 2.42.1), I follow the recommandation :
> It is highly recommended to start with new/fresh settings, especially because of the change in audio handling! You can avoid unwanted side effects by not using or copying your old settings over.

But I would like to know what is the recommanded way to import/export encoder profiles ? As I presume it would be affected by major changes with 2.42.0, I can normally reuse them like my previous version. But I don't find a way to export it.

So I guess the solutions are :
- doing it from scratch and changing options and adding CLI commands one by one
- copy command line from previous version and import command line for actual version (but it seems not exactly changing it 1:1, maybe because of new default's settings ?)
- copying old settings.bat  and modifying after this the non-encoder profiles options (probably not recommanded)

Which one is recommanded ?
Regards.

**Comment by troxxy797** [comment]
Do it from scratch, it's explicitly mentioned in version 2.42 and above. Doesn't take that long, 1 minute max, trust me.

**Comment by pokemaster974** [comment]
> Do it from scratch, it's explicitly mentioned in version 2.42 and above. Doesn't take that long, 1 minute max, trust me.

I don't know how you count 1 min (especially if there are multiple encoding profiles), but ok.

**Comment by Dendraspis** [comment]
Depending on the configuration it really can take some time....sadly.  😞 

> As I presume it would be affected by major changes with 2.42.0, I can normally reuse them like my previous version. But I don't find a way to export it.

Just to make it clear: Once `v2.42.0` is set up, you don't have to reset any settings when updating to `v2.42.x`.
Historically grown there is no better way to transfer the them by hand and, at least for video encoders, use the `Import Command Line` feature. Some parameters are sort of special and it's a good advice to double-check the command line after importing. Sometimes default settings have changed, but sometimes there is a specific bug related to a parameter. In that case please report it.
I also recommend starting the old and new version in parallel next to each other and go through the settings. This has also the advantage, that you notice directly changes, for example regarding AutoCrop, which is always a good start into a new version.

> copying old settings.bat and modifying after this the non-encoder profiles options (probably not recommanded)

With `v2.42` and it's new audio implementation it will definitely cause issues as some others have already reported. Usually you can do it, but it may cause side effects, sometimes pretty strange ones. So if it works for you, go for it. But if you encounter issues, please try starting again from scratch with fresh settings by simply renaming the `Settings` folder, so you can easily jump back. Especially before you open a bug report here on GitHub. 😅

**Comment by Dendraspis** [comment]
Another issue with the same topic I have already answered with partly more information:  #1475
